data/campaigns/Eastern_Invasion/utils/items.cfg

```diff
@@ -0,0 +1,1193 @@
+<span size='small'><i>  Picking up an item consumes your moves and attack.</i></span>
+<b></b>"
+#enddef
+
+#-------------------
+# CRYSTAL QUIVER
+#-------------------
+# anybody can wield this, but you only benefit from the "quiver" part if you have a bow/crosbow
+#define ID_CRYSTAL_QUIVER
+crystal_quiver #enddef
+
+#define PLACE_ITEM_CRYSTAL_QUIVER X Y
+    {PLACE_ITEM
+    {ID_CRYSTAL_QUIVER}  _"quiver"  items/crystal-quiver.png  {X}  {Y}  ()
+    _"Crystal Quiver"
+    _"Arrows from this crystalline quiver glimmer with a pale magical light, <i><b>illuminating</b></i> the surrounding area and making your bow or crossbow attacks <i><b>arcane</b></i>."
+    {NOTE_REUSEABLE}
+    ""
+    ([effect]
+    apply_to=attack
+    range=ranged
+    set_type=arcane
+    name=bow,crossbow,composite bow
+[/effect]
+[effect]
+    apply_to=new_ability
+    [abilities]
+        {ABILITY_ILLUMINATES}
+    [/abilities]
+[/effect]
+[effect]
+    apply_to,halo=halo,halo/illuminates-aura.png
+[/effect]
+[effect]
+    apply_to,add=overlay,misc/crystal-quiver-icon.png
+[/effect])
+()
+}
+#enddef
+
+#define GLOBAL__DROPPABLE_CRYSTAL_QUIVER
+    {GLOBAL__DROPPABLE
+    {ID_CRYSTAL_QUIVER}
+    items/crystal-quiver.png
+    _"Drop Crystal Quiver"
+    ({PLACE_ITEM_CRYSTAL_QUIVER $unit.x $unit.y})
+    ({PLACE_ITEM_CRYSTAL_QUIVER $x1 $y1})
+    }
+#enddef
+
+#-------------------
+# HOLY AMULETS
+#-------------------
+#define ID_HOLY_AMULET_1
+holy_amulet_1 #enddef
+
+#define ID_HOLY_AMULET_2
+holy_amulet_2 #enddef
+
+#define ID_HOLY_AMULET_3
+holy_amulet_3 #enddef
+
+#define PLACE_ITEM_HOLY_AMULET ID X Y
+    {PLACE_ITEM
+    {ID}  _"amulet"  items/ankh-necklace.png  {X}  {Y}  ()
+    _"Holy Amulet"
+    _"Engraved with a consecrated symbol, this amulet will bless both your <i><b>melee</b></i> and <i><b>ranged</b></i> attacks with <i><b>arcane</b></i> damage."
+    {NOTE_REUSEABLE}
+    ""
+    ([effect]
+    apply_to=attack
+    set_type=arcane
+[/effect]
+[effect]
+    apply_to=overlay
+    add=misc/arcane-icon.png
+[/effect])
+()
+}
+#enddef
+
+#define GLOBAL__DROPPABLE_HOLY_AMULET ID
+    {GLOBAL__DROPPABLE
+    {ID}
+    items/ankh-necklace.png
+    _"Drop Holy Amulet"
+    ({PLACE_ITEM_HOLY_AMULET {ID} $unit.x $unit.y})
+    ({PLACE_ITEM_HOLY_AMULET {ID} $x1 $y1})
+    }
+#enddef
+
+#-------------------
+# SHIELD OF THE SENTINEL
+#-------------------
+#define ID_SENTINEL
+sentinel #enddef
+
+#define PLACE_ITEM_SHIELD_OF_THE_SENTINEL X Y
+    {PLACE_ITEM
+    {ID_SENTINEL}  _"sentinel"  items/shield-of-the-sentinel.png  {X}  {Y}  ()
+    _"Shield of the Sentinel"
+    _"Deep within this shield’s towering bulk, enchanted machinery whirrs faintly. Whenever an adjacent ally is hit by an attack, <i><b>this shield’s bearer is hit instead</b></i>."
+    {NOTE_REUSEABLE}
+    ""
+    ([effect]
+    apply_to=overlay
+    add=misc/sentinel-icon.png
+[/effect])
+([remove_item]
```

Comment:
Looks like both indentation and WML checks can't handle this convention of placing WML inside brackets without any spacing. Perhaps separate them (in lieu of indentation being improved)?